controllers/scripts.go

```diff
@@ -61,22 +70,44 @@ ipfs config Addresses.API /ip4/0.0.0.0/tcp/5001
 ipfs config Addresses.Gateway /ip4/0.0.0.0/tcp/8080
 ipfs config --json Swarm.ConnMgr.HighWater 2000
 ipfs config --json Datastore.BloomFilterSize 1048576
-ipfs config Datastore.StorageMax 100GB
+ipfs config Datastore.StorageMax {{ .StorageMax }}GB
 
 chown -R ipfs: /data/ipfs
 `
 )
 
 func (r *IpfsReconciler) configMapScripts(m *clusterv1alpha1.Ipfs, cm *corev1.ConfigMap) (controllerutil.MutateFn, string) {
 	cmName := "ipfs-cluster-scripts-" + m.Name
+	configureTmpl, _ := tmpl.New("configureIpfs").Parse(configureIpfs)
+	var storageMaxGB string
+	parsed, err := resource.ParseQuantity(m.Spec.IpfsStorage)
```

Comment:
Is `IpfsStorage` user-provided?

It it is I'm not sure we want to magically set 80% of it.

I wonder though, if we can get the size of the volume attached from kubernetes, and use that to auto-configure.

Reply:
Yeah this is user-defined, but this is what is used to create to create the volumes. What's a better formula to use for this?

Reply:
@hsanjuan We can absolutely get the size of the volume in Kubernetes, are you thinking of the user providing their own volume and then passing it into the IPFS Cluster spec, or something else?